core-plugins/docs/SSHAction-action.md

```diff
@@ -21,9 +21,9 @@ Properties
 
 **user:** The username used to connect to host.
 
-**privateKeyFile:** The file path to Private key.
+**privateKeyFileLookupKey:** The file path to Private key.
```

Comment:
Also, can this parameter be named better? Seems very verbose for something that's exposed to users. What does it do? Also seems like it could use some more detailed docs.